src/amber/router/pipe/flash.cr

```diff
@@ -106,6 +106,15 @@ module Amber
           self
         end
 
+        def merge!(other : Hash(String, String)?)
```

Comment:
yes, I clean up. Since the flash hash could be nil to avoid the check on the other side. Should be cleaned now